File: Ejercicio 27/ej27.js
let n = 920

function get_dates(d){
    let años = Math.floor(d / 365);
    //console.log(`años: ${años}`);
    let remmaning_days = d % 365;
    let meses = Math.floor(remmaning_days / 30)
    //console.log(meses);
    let days = remmaning_days % 30
    console.log(`años: ${años} meses: ${meses} días: ${days}`);
} 

get_dates(n);

File: Ejercicio 27/ej27.py
days = 920

def calculate_days(d):
    years = d // 365
    d = d % 365
    print(f"años: {years}", end=" ")
    months = d // 30
    print(f"meses: {months}", end=" ")
    d = d % 30
    print(f"días: {d}", end=" ")

calculate_days(days)